RELEASE-NOTES.rst

```diff
@@ -18,6 +18,220 @@
+
+Public interface
+~~~~~~~~~~~~~~~~
+
+-  Keeps the active tab on the document or person detailed view on page
+   reloads.
+
+Professional interface
+~~~~~~~~~~~~~~~~~~~~~~
+
+-  Renders the language menu with the same look as in the public
+   interface:
+
+   -  Uses the same icons.
+   -  Removes the current language.
+   -  Avoids to translate the language menu.
+   -  Adds a divider to separate the language menu from the help link.
+
+-  Takes the entire screen width. This is useful for the improvement of
+   the editor.
+-  Improves the document editor:
+
+   -  Sets a max width for selects and inputs.
+   -  Sets bold font weight for titles.
+   -  Hides fields with unique value (as ``bf:Place``).
+   -  Displays fields inline and makes this configurable through a HTML class
+      in the JSON schema.
+   -  Adds HTML class in the JSON schema to fix max width and title font
+      size.
+
+-  Implements interface components to import bibliographic records from
+   external sources through the web (SRU protocol). The librarian searches in a
+   regular search interface for the desired record, using a simple query
+   (keywords for the author, title, date, IDs…), selects a record, gets a
+   preview in two formats (the RERO ILS JSON rendered in HTML, and MARC). Once
+   the desired record is identified, through an *Import* button, the record is
+   loaded into the document editor to be modified and then added to the
+   catalog. At this stage, only the BnF SRU service is implemented.
+-  Fixes the redirection to the parent document after the deletion of an item.
+   This behaviour has been generalized to every parent/child resource
+   relationship through a modification in the routing configuration.
+-  Allows to receive an issue (serials) through the professional
+   interface (see the acquisition section, below). The workflow begins on the
+   holding detailed view and then continues on a dedicated view for serial
+   issues. The *Quick receipt* avoids the burden to manually create an item,
+   when possible.
+-  Improves the transaction data displayed on the item detailed view,
+   depending on the transaction type (checkout or request), to avoid
+   displaying an empty pickup location name when this data is not
+   relevant.
+
+Circulation
+-----------
+
+-  Adapts the patron profile URL in notification messages to the active RERO
+   ILS instance.
+-  Adds a CLI for notifications to start the notification process with
+   ``invenio run notifications process``.
+
+Metadata
+--------
+
+-  Adds a translation mechanism for the resource editor (documents, items…).
+   The translated schemas keys in the `rero-ils` project are served through an
+   API endpoint to the `rero-ils-ui` project, in order to avoid translating
+   them twice.
+-  Holding record, serial pattern:
+
+   -  Adds a field to describe the publication frequency of a serial pattern.
+      The librarian has to pick from a fixed list of 15 periodicities, which
```

Comment:
`15 frequencies` ?
+ reference @ http://www.rdaregistry.info/termList/frequency/

Reply:
periodicities is a synonym. That way, we avoid to repeat the same word in the sentence.